fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -4,26 +4,6 @@ import NeedsExistingManifestHelpers from '../../mixins/needs-existing-manifest-h
 
 export default Ember.Route.extend(NeedsExistingManifestHelpers, {
 
-  beforeModel() {
-    // Ensure the models have been persisted so that we're validating/syncing up to date data.
-    let deployment = this.modelFor('deployment');
-
-    if (deployment.get('isStarted')) {
-      return;
-    }
-
-    let promises = {
-      deployment: deployment.save()
-    };
-
-    if (deployment.get('deploy_openstack')) {
-      promises.openstack_deployment = deployment.get('openstack_deployment')
-        .then(openstack_deployment => openstack_deployment.save());
-    }
-
-    return Ember.RSVP.hash(promises);
-  },
```

Comment:
@cfchase, @eriknelson, this `beforeModel` caused an error. Can you please check if there's a need to still have this using ember-data 2.4?